### PR TITLE
Add support for global tags

### PR DESCRIPTION
**Pull request by mpiercy827**
This PR adds the ability to configure Statix with global tags, fixing [#13](https://github.com/lexmag/statix/issues/13).

### CODE REVIEW

**Comment by mpiercy827** [comment]
Just want to bump this, as it would be great for our team to have!

File: lib/statix.ex [hunk 1]
```diff
@@ -365,4 +367,18 @@ defmodule Statix do
...
+    global_tags = Application.get_env(:statix, :tags)
+    options_tags = Keyword.get(options, :tags)
+
+    new_tags = cond do
+      options_tags && global_tags -> options_tags ++ global_tags
+      global_tags -> global_tags
+      true -> nil
+    end
+
+    if new_tags, do: Keyword.put(options, :tags, new_tags), else: options
```
Comment (whatyouhide) [inline lib/statix.ex]: Not sure if @lexmag would agree to this but if so I think a better implementation would be:

```elixir
global_tags = Application.get_env(:statix, :tags, [])
Keyword.update(options, :tags, [], &Keyword.merge(global_tags, &1))
```

However you need to handle global backend-specific tags, that is, what would be configured by

```elixir
config :statix, MyApp.Statix,
  tags: ...
```

**Comment by mpiercy827** [comment]
@whatyouhide I've cleaned up the implementation a bit and added support for backend-specific global tags.

**Comment by mpiercy827** [comment]
Want to bump this again!

**Comment by uberbrodt** [comment]
I'm interested in this as well! Anything I can do to help get this out?

**Comment by chiragtoor** [comment]
I could use this as well, if there is anything left to be done I can try and help out.

**Comment by mpiercy827** [comment]
@whatyouhide Wanted to bump this again, as it would be a huge help for us!

**Comment by minhajuddin** [comment]
@lexmag Is there anything pending for this to be merged :slightly_smiling_face: . This would be a great addition to most users.

**Comment by hylkealons** [comment]
Bumping it again because it's a couple of months later. Can we get this merged @lexmag?

**Comment by lexmag** [comment]
Sorry folks for keeping you waiting, the PR will be merged during the week (after Travis CI migrated the repository).

**Comment by lexmag** [comment]
Thank you @mpiercy827. 💛

**Pull request merged**